### PR TITLE
Reduce spacing between collections on the CollectionList component

### LINKED ISSUE #5452: The distance between collection rows in My Collections page could be reduced
Follow up for https://github.com/mozilla/addons-frontend/issues/3142

STR:
1. Log in to AMO -dev
2. Open My Collections page https://addons-dev.allizom.org/en-US/firefox/collections/
3. Observe the page layout 

Actual result:
There is too much white space between collection rows

Expected result:
White space between collection names should be reduced in order to make My Collections page look more compact

Notes:
- reproduced on AMO-dev with FF60, WIn10x64

image

### PR DESCRIPTION
**Pull request by bobsilverberg**
Fixes #5452 

Before:

screenshot 2018-07-05 16 28 01

After:

screenshot 2018-07-05 16 27 31

### CODE REVIEW

**Comment by bobsilverberg** [comment]
Thanks @rebmullin. I really should have checked that first. :-(

I've updated it to look much better on mobile. Please take another look.

**Comment by rebmullin** [comment]
mobile looks better now : ) but I want to make sure for desktop you intended this:

Alt text

it seems that spacing between name and number is more than it was before...is that intended?

also, I am wondering if it's easier to just put the padding on the mobile and then remove it for desktop. for example:

```css
.CardList ul > li.UserCollection {
  padding: 5px $padding-page-l;

   @include respond-to(large) { // over whatever breakpoint this should be
      padding-top: 0;
      padding-bottom: 0;
   }
}
```

not required, just a suggestion :)

**Comment by bobsilverberg** [comment]
> it seems that spacing between name and number is more than it was before...is that intended?

No, that's not intended. Let me try again...

**Comment by bobsilverberg** [comment]
Thanks @rebmullin, that css change you suggested is also a big improvement. I believe I have resolved the latest desktop issue. Do you mind taking another look?

**Comment by bobsilverberg** [comment]
Thanks again for pointing that out @rebmullin. I'm sorry for not checking things so thoroughly. I actually noticed another issue with the mobile version while I was trying to fix the last item - the card didn't have rounded corners at the bottom. But I've fixed that now, and I think everything looks good on both sizes now.

Please do take a final look though. :-) Thanks!

**Comment by rebmullin** [comment]
agreed : ) everything is looking good now 🚀
